Translate/translate_table.py

```diff
@@ -141,28 +148,28 @@ def parse_regex_flags(flags_num: int) -> str:
     "Jump": lambda obj: add_jump_blocks(obj, "Jump") or "",
     "JumpLoop": lambda obj: add_jump_blocks(obj, "JumpLoop") or "",
     "JumpIfTrue": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU)",
-    "JumpIfFalse": lambda obj: add_jump_blocks(obj, "If") or "if (!ACCU)",
+    "JumpIfFalse": lambda obj: add_jump_blocks(obj, "If") or "if (!(ACCU))",
     "JumpIfNull": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU == null)",
     "JumpIfNotNull": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU != null)",
     "JumpIfUndefined": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU == undefined)",
     "JumpIfNotUndefined": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU != undefined)",
     "JumpIfUndefinedOrNull": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU == undefined)",
-    "JumpIfToBooleanTrue": lambda obj: add_jump_blocks(obj, "If") or "if (ACCU)",
-    "JumpIfToBooleanFalse": lambda obj: add_jump_blocks(obj, "If") or "if (!ACCU)",
+    "JumpIfToBooleanTrue": lambda obj: add_jump_blocks(obj, "If") or "if (boolean(ACCU))",
+    "JumpIfToBooleanFalse": lambda obj: add_jump_blocks(obj, "If") or "if (!boolean(ACCU))",
```

Comment:
To distinguish between `JumpIfTrue/False` and `JumpIfToBooleanTrue/False` there should be a `boolean` wrapper, because in an opposite case V8 would use the same operator `JumpIfTrue/False` for everything.